mlflow/tracing/types/model.py

```diff
@@ -22,6 +22,12 @@ class Trace:
     def to_json(self) -> str:
         return json.dumps(asdict(self), default=str)
 
+    def _repr_mimebundle_(self, include=None, exclude=None):
```

Comment:
Can we write a docstring for this explaining how it's used?

Reply:
added, and also changed `text/plain` to a more helpful representation so it doesn't look bad in non-db notebooks